src/lc0ctl/leela2onnx.cc

```diff
@@ -40,6 +43,16 @@ const OptionId kInputFilenameId{"input", "InputFile",
                                 "Path of the input Lc0 weights file."};
 const OptionId kOutputFilenameId{"output", "OutputFile",
                                  "Path of the output ONNX file."};
+const OptionId kHloTextOutputFilenameId = {
+    "hlo-text-output", "HloTextOutputFile", "Path of the output HLO file."};
```

Comment:
Why do we add uci names for the options? Just curious.

Reply:
No reason. I thought it wouldn't harm, but actually they show up in `--help`, so better remove them. (Done)